src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -549,10 +555,9 @@ object Parameters {
       */
     def setColumnMap(fromSchema: Option[StructType],
                      toSchema: Option[StructType]): Unit = {
-      assert(
-        parameters.get(PARAM_COLUMN_MAP).isEmpty,
-        "Column map is already declared"
-      )
+      if (parameters.get(PARAM_COLUMN_MAP).isDefined) {
+        throw new Exception("Column map is already declared")
+      }
```

Comment:
What is the difference between Assert and Exception? Do you want to break the process or not?

Reply:
IMO, Assert should not be used in production code.